W-18139363: (W-18122696 duke edit) Update the Anypoint Monitoring Hybrid installation docs for Otel release (#363)
mulesoft/docs-monitoring: modules/ROOT/pages/monitoring-settings-page.adoc

include::partial$include-nav-steps-no-permissions.adoc[]

. In the Anypoint Monitoring navigation menu, click *Settings*. 
. To install Anypoint Monitoring to monitor applications deployed to a local server, select *Hybrid*. 
+

> The UI displays instructions for installing Anypoint Monitoring on-premises. The UI provide access to the latest version of the Anypoint Monitoring agent (`am.zip`). For earlier versions, you must download the agent from the *Anypoint Monitoring* tab in your customer portal. 

Author: ```suggestion
The UI displays instructions for installing Anypoint Monitoring on-premises. The UI provides access to the latest version of the Anypoint Monitoring agent (`am.zip`). 
//HIDING FOR NOW:
// For earlier versions, you must download the agent from the *Anypoint Monitoring* tab in your customer portal. 
```

> To use an earlier version, see the knowledge article https://help.mulesoft.com/s/downloads?tabset-169e6=7ea44[Unable to Download Anypoint Monitoring Agent Installer for Windows after April 1st, 2025 from Anypoint Monitoring UI]. 

Author: ```suggestion
//TODO: TEMPORARY UNTIL WE have a better location for earlier versions
To use an earlier version, see the knowledge article https://help.mulesoft.com/s/downloads?tabset-169e6=7ea44[Unable to Download Anypoint Monitoring Agent Installer for Windows after April 1st, 2025 from Anypoint Monitoring UI]. 
```

include::partial$include-nav-steps-no-permissions.adoc[]

. In the Anypoint Monitoring navigation menu, click *Settings*. 

> . To install Anypoint Monitoring to monitor applications deployed to a local server, select *Hybrid*. 

Author: ```suggestion
. To monitor applications deployed to a local server, select *Hybrid*, and follow the procedure to install Anypoint Monitoring agent from the latest version of the agent's installer in `am.zip`. 
```

> +
> //HIDING FOR NOW:
> // For earlier versions, you must download the agent from the *Anypoint Monitoring* tab in your customer portal. 

Author: ```suggestion
+
To use an earlier version, see the knowledge article https://help.mulesoft.com/s/downloads?tabset-169e6=7ea44[Unable to Download Anypoint Monitoring Agent Installer for Windows after April 1st, 2025 from Anypoint Monitoring UI]. 
```

> To use an earlier version, see the knowledge article https://help.mulesoft.com/s/downloads?tabset-169e6=7ea44[Unable to Download Anypoint Monitoring Agent Installer for Windows after April 1st, 2025 from Anypoint Monitoring UI]. 

Author: ```suggestion
To use an earlier version, see the knowledge article https://help.salesforce.com/s/articleView?id=004576938&type=1[Unable to Download Anypoint Monitoring Agent Installer for Windows after April 1st, 2025 from Anypoint Monitoring UI]. 
```

include::partial$include-nav-steps-no-permissions.adoc[]

. In the Anypoint Monitoring navigation menu, click *Settings*. 
. To install Anypoint Monitoring to monitor applications deployed to a local server, select *Hybrid*. 
+

> The UI displays instructions for installing Anypoint Monitoring on-premises. The UI provides access to the latest version of the Anypoint Monitoring agent (`am.zip`). 

Reviewer: ```suggestion
```
